src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -81,7 +81,7 @@ public String getContainer() {
     }
 
     @DataBoundSetter
-    public void setContainer(String container) throws IOException {
+    public void setContainer(String container) throws Failure {
```

Comment:
Why just on this one and not the others? Does not really matter, but is potentially confusing.